docs/data/toolpad/core/pages.ts

```diff
@@ -85,7 +85,7 @@ const pages: MuiPage[] = [
           },
           {
             pathname: '/toolpad/core/react-persistent-state',
-            title: 'Peristent State',
+            title: 'Peristent storage state',
```

Comment:
1. typo in "peristent" => I make that "Persisted"
1. we're planning to put url query state under this umbrella as well, so I'd leave out "storage"

```suggestion
            title: 'Persisted state',
```